### DISCUSSION TITLE
How to use IDDPM in diffusers ?

### DISCUSSION THREAD

**Discussion by jiarenyf** [huggingface/diffusers#7721] [General]
The code base is here:
https://github.com/openai/improved-diffusion/blob/main/improved_diffusion/gaussian_diffusion.py

**Comment by tolgacangoz** [comment]
`diffusers` supports [learnable variance](https://huggingface.co/docs/diffusers/main/en/api/schedulers/ddpm#diffusers.DDPMScheduler.variance_type) for inferencing.
